Bug in Dockerfile

When you follow the README to create a new blog instance on a fresh environment
At the `make start-image` step we encounter a compilation error but we cannot see it because by default the run command is daemonized.

In order to reproduce the bug, you have to remove the `-d` option, so just run:

`docker run -t -p 3000:3000 --rm --name sogilis_blog blog/hugoasciidoc`

Then we the the error:

```bash
docker run  -t -p 3000:3000 --rm --name sogilis_blog blog/hugoasciidoc

> victor-hugo@1.0.0 start /blog
> gulp server

[15:17:17] Failed to load external module @babel/register
[15:17:17] Requiring external module babel-register
[15:17:18] Using gulpfile /blog/gulpfile.babel.js
[15:17:18] Starting 'hugo'...
[15:17:18] Starting 'css'...
[15:17:20] Starting 'js'...
[15:17:20] Starting 'svg'...
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: spawn ./bin/hugo.linux ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:229:19)
    at onErrorNT (internal/child_process.js:406:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:235:12)
    at onErrorNT (internal/child_process.js:406:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! victor-hugo@1.0.0 start: `gulp server`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the victor-hugo@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-10-24T15_17_20_507Z-debug.log
Makefile:9: recipe for target 'start-image' failed
make: *** [start-image] Error 1
```

**Expected behavior**

It shall work

--- a/Dockerfile
+++ b/Dockerfile
@@ -4,7 +4,8 @@ WORKDIR /blog
 
 RUN apk add --no-cache asciidoctor wget
 
-COPY ./bin/hugo.linux /bin/hugo
+COPY ./bin/hugo.linux ./bin/hugo.linux
+RUN chmod a+x ./bin/hugo.linux
 COPY site site
 COPY src src
 COPY .*rc ./

